.dockerignore

```diff
@@ -0,0 +1 @@
+frontend/node_modules
```

Comment:
nit: no newline at end of file :-) https://unix.stackexchange.com/a/18789/250345